x-pack/filebeat/processors/decode_cef/cef/testdata/samples.log

```diff
@@ -0,0 +1,14 @@
+CEF:0|security|threatmanager|1.0|100|trojan successfully stopped|10|src=10.0.0.192 additional.dotfieldName=new_value ad.Authentification=MICROSOFT_AUTHENTICATION_PACKAGE_V1_0 ad.Error_,Code=3221225578 dst=12.121.122.82 ad.field[0]=field0 ad.foo.name[1]=new_name
```

Comment:
It would be great if we had a corresponding JSON file to this one, to intg test (and review) the mapping of this broader set of events, like we have in the Filebeat modules

Reply:
I have added a golden file that contains the output of this processor on these samples.

There will be a Filebeat module that uses this processor. I will create this after merging the processor. That module will have tests too.